Cirrious.FluentLayout/FluentLayoutExtensions.cs

```diff
@@ -18,8 +18,8 @@ public static void SubviewsDoNotTranslateAutoresizingMaskIntoConstraints(this UI
             foreach (var subview in view.Subviews)
             {
                 subview.TranslatesAutoresizingMaskIntoConstraints = false;
-		
-	        if (subview.Subviews != null && subview.Subviews.Count() > 0)
+
+                if ((subview as UIControl) == null && subview.Subviews != null && subview.Subviews.Count() > 0)
```

Comment:
Why do you cast to UIControl? You can't put anything else than subclasses of UIView into the subview collection and far from all controls will be UIControl...